lametro/templates/partials/board_report_statuses.html

```diff
@@ -0,0 +1,29 @@
+<table class="table">
+    <thead>
+        <tr>
+            <th>Status</th>
+            <th class="desktop-only">Description</th>
+        </tr>
+    </thead>
+    <tbody>
+    {% for status in BILL_STATUS_DESCRIPTIONS %}
```

Comment:
I know I proposed adding this to `city_context` but it occurs to me that we won't have this info for the other instances. What do you think about scratching the change to `django-councilmatic` and passing this variable directly into the Metro about page context?